queue_services/entity-bn/requirements.txt

```diff
@@ -11,6 +11,7 @@ certifi==2020.12.5
 click==8.1.3
 dpath==2.0.1
 itsdangerous==1.1.0
+nest_asyncio
```

Comment:
What's this for?

Reply:
split dependency in legal-api (https://github.com/bcgov/lear/commit/5a01a1ff693197fc958e6d94d8bbc829366dfc66) which cause error